web/client-api/src/test/java/io/deephaven/web/ClientIntegrationTestSuite.java

```diff
@@ -28,6 +29,7 @@ public static Test suite() {
         suite.addTestSuite(NullValueTestGwt.class);
         suite.addTestSuite(HierarchicalTableTestGwt.class);
         suite.addTestSuite(PartitionedTableTestGwt.class);
+        suite.addTestSuite(LazyPromiseTestGwt.class);
```

Comment:
Add a comment on this that it isn't really an integration test?

Reply:
I had put a comment in `LazyPromiseTestGwt#getModuleName` itself, but makes sense to have here as well.